modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/activity/WebViewActivity.kt

```diff
@@ -64,6 +62,20 @@ class WebViewActivity : AppCompatActivity(), CoroutineScope {
         setSupportActionBar(binding.toolbar)
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
 
+        onBackPressedDispatcher.addCallback(
+            this,
+            object : OnBackPressedCallback(true) {
+                override fun handleOnBackPressed() {
+                    if (binding.webview.canGoBack()) {
+                        binding.webview.goBack()
+                    } else {
+                        isEnabled = false
```

Comment:
What does `isEnabled = false` do?

Reply:
This is to disable the custom back pressed after using it. I have seen some examples using like this, but I've just simplified the code